diesel_compile_tests/tests/compile-fail/must_use_query_methods.rs

```diff
@@ -26,5 +26,14 @@ fn main() {
     st.filter(b.eq(true)); //~ ERROR unused `diesel::query_builder::SelectStatement`
     st.filter(b.eq(true)).limit(1); //~ ERROR unused `diesel::query_builder::SelectStatement`
 
+    insert_into(stuff)
+        .values(b.eq(true)); //- ERROR unused `diesel::query_builder::InsertStatement`
```

Comment:
I'd just do `ERROR unused` here, and have fewer lines to change if the displayed path changes.